src/blockchain_txn_poc_receipts_v2.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package helium;
+
+import "blockchain_poc_core_v1.proto";
+
+message blockchain_txn_poc_receipts_v2 {
```

Comment:
If this is the same as receipts_v1 why are we duplicating it

Reply:
the data type for field 7 (request_block_hash vs. block_hash) is the same but will they be handled by the encoding/decoding code sufficiently differently as to justify a different versioned pb message?

Reply:
i believe we need to trash all open/active PoC txns that are mid-flight when the chain vars are switched over so we wouldn't have to worry about a txn being encoded by one code path and decoded by another so would it be sufficient to have the code generating/receiving these txns handle that last field appropriately based _solely_ on the chain vars? (i.e. look up a requesting block by hash vs. a challenge keys submission block by hash)